applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_line_interface_element.cpp

```diff
@@ -0,0 +1,544 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_constitutive/incremental_linear_elastic_interface_law.h"
+#include "custom_elements/line_interface_element.h"
+#include "custom_geometries/line_interface_geometry.h"
+#include "geo_mechanics_application_variables.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+#include <cstddef>
+
+namespace
+{
+
+using namespace Kratos;
+
+PointerVector<Node> CreateNodes()
+{
+    PointerVector<Node> result;
+    result.push_back(Kratos::make_intrusive<Node>(1, 0.0, 0.0, 0.0));
+    result.push_back(Kratos::make_intrusive<Node>(2, 1.0, 0.0, 0.0));
+    result.push_back(Kratos::make_intrusive<Node>(3, 0.0, 0.0, 0.0));
+    result.push_back(Kratos::make_intrusive<Node>(4, 1.0, 0.0, 0.0));
+
+    return result;
+}
+
+std::shared_ptr<Properties> CreateLinearElasticMaterialProperties(double normal_stiffness, double shear_stiffness)
```

Comment:
When following the Kratos Style Guide we should use the following names:
```suggestion
std::shared_ptr<Properties> CreateLinearElasticMaterialProperties(double NormalStiffness, double ShearStiffness)
```